examples/websockets/client.js

```diff
@@ -0,0 +1,18 @@
+var sharedb = require("sharedb/lib/client");
+
+var connection = new sharedb.Connection(new WebSocket("ws://localhost:8888"));
+
+var doc = connection.get("collection", "document");
+
+doc.subscribe(function(error){
+  if (!doc.type) {
+    doc.create({x: "hello"});
+  }
+
+  if (doc.type) {
+    // `doc` ready, make ops
+    setTimeout(function() { // timeout need for example (to delay after create)
+      doc.submitOp([{p: ["x", 5], si: " world"}]);
+    }, 1000);
+  }
+});
```

Comment:
Would be great to add a listener for `doc.on('op')` to show how that event works.

(Also, mega nitpick: add newlines at the end of the files!)